src/System.Security.Cryptography.Pkcs/src/MatchingRefApiCompatBaseline.netstandard.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Security.Cryptography.Pkcs:
+MembersMustExist : Member 'System.Security.Cryptography.Pkcs.SignerInfo.GetSignature()' does not exist in the implementation but it does exist in the contract.
```

Comment:
@bartonjs should these APIs be in the reference assembly?

Reply:
> What is actually being diffed here?

Oh, right, the filename says `netstandard`.  These members don't belong in netstandard because of type unification with netfx (the members are new in net472).

I can make them be declared internal for netstandard, or leave the baseline; whichever model we're utilizing elsewhere.

Reply:
This baseline is for the netstandard build. It found these in the netstandard implementation but not in the netstandard ref.

Reply:
Ideally we fix the implementation if we can to avoid the need for the baseline but if we cannot for some reason we can keep the baseline.

Reply:
@bartonjs Please take a look at https://github.com/dotnet/corefx/pull/27881/commits/5ec673521f9abea16a1dc88f5236f93c008fae34 where I updated the crypto libraries.